components/pkg-export-kubernetes/src/manifest.rs

```diff
@@ -61,7 +61,9 @@ impl Manifest {
         );
         let ring_secret_name = matches.value_of("RING_SECRET_NAME").map(|s| s.to_string());
         // clap ensures that we do have the mandatory args so unwrap() is fine here
-        let pkg_ident_str = matches.value_of("PKG_IDENT_OR_ARTIFACT").unwrap();
+        let pkg_ident_str = matches.value_of("PKG_IDENT_OR_ARTIFACT").expect(
```

Comment:
Why is this its own commit? The previous one seems to handle the same situation.